scripts/pyenv_setup.sh

```diff
@@ -37,21 +37,22 @@ get_shell_startup_script() {
   echo "$_startup_script"
 }
 
-# The first \n is important on Github workers sicne it was being appended to
+# The first \n is important on Github workers since it was being appended to
 # the last line rather than on a new line. I never figured out why
 _append_to_startup_script() {
   if [[ -n "$SHELL" ]]; then
     case "$SHELL" in
     */bash)
       # shellcheck disable=SC2016
-      echo -e '\nif command -v pyenv 1>/dev/null 2>&1; then\n  eval "$(pyenv init --path)"\nfi' >>"${1}"
+      echo -e '\neval "$(pyenv init --path)"\nif command -v pyenv 1>/dev/null 2>&1; then\n  eval "$(pyenv init -)"\nfi' >>"${1}"
       ;;
     */zsh)
       # shellcheck disable=SC2016
-      echo -e '\nif command -v pyenv 1>/dev/null 2>&1; then\n  eval "$(pyenv init --path)"\nfi' >>"${1}"
+      echo -e '\neval "$(pyenv init --path)"\nif command -v pyenv 1>/dev/null 2>&1; then\n  eval "$(pyenv init -)"\nfi' >>"${1}"
       ;;
     */fish)
-      echo -e '\n\n# pyenv init\nif command -v pyenv 1>/dev/null 2>&1\n  pyenv init --path | source\nend' >>"$1"
+      # shellcheck disable=SC2016
+      echo -e '\n\n# pyenv init\nstatus is-login; and pyenv init --path | source\nif command -v pyenv 1>/dev/null 2>&1;\n  pyenv init - | source\nend' >>"$1"
```

Comment:
I figured since `pyenv init --path` simply sets an environment variable it would be safe to run it without checking the output of `pyenv`. `pyenv init -` in comparison does a lot more and it loosely relies on pyenv being present and potentially working (see: [these lines in pyenv, where `path` is `init --path` and `print` is `init -`](https://github.com/pyenv/pyenv/blob/fadb3f33896991a61348649b350bf9c31b42dd38/libexec/pyenv-init#L58-L69)) so I kept the `command -v` check in all three shells.

Reply:
The line is getting a little too long:
```suggestion
      echo -e '\n\n# pyenv init\nstatus is-login; and pyenv init --path | source' \
              '\n if command -v pyenv 1>/dev/null 2>&1;\n  pyenv init - | source\nend' >>"$1"
```